RootOutputer.cc

```diff
@@ -20,6 +21,7 @@ RootOutputer::RootOutputer(std::string const& iFileName, unsigned int iNLanes, C
   basketSize_{iConfig.basketSize_},
   splitLevel_{iConfig.splitLevel_}
 {
+  TFileCacheWrite(&file_, iConfig.cacheSize_);
```

Comment:
This has the `new` missing.

Reply:
Also, please make this optional by using a new configuration option to turn this one. The default of the new option should be so this is off.

Reply:
Actually, just using `cacheSize_>0` is good enough. No need for another configuration parameter.

Reply:
Sorry. I pushed from the mirror again.